src/components/SecondaryPanes/Expressions.css

```diff
@@ -6,12 +6,20 @@
   cursor: pointer;
   color: var(--theme-body-color);
   background-color: var(--theme-body-background);
-  line-height: 12px;
+  font-size: 12px;
 }
 
 .input-expression::-webkit-input-placeholder {
   text-align: center;
   font-style: italic;
+  color: var(--theme-body-color);
+}
+
+.input-expression::-moz-placeholder {
```

Comment:
learned something today ! Hopefully chrome will have the unprefixed version soon too.